src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -224,6 +236,10 @@ The URL is an identifier only. OpenSearch Dashboards and your browser will never
       autosize = defaultAutosize;
     }
 
+    if (this.showEvents) {
+      autosize = showEventsAutosize;
```

Comment:
Currently, with the way that the vega spec is generated from a line chart config, the L243 conditional isn't possible, since `this.spec.width` and `this.spec.height` are never set, and will always default to `autosize` being either `defaultAutosize` or `showPointInTimeEventsAutosize`. It would only be truthy given a user-created spec from the vega chart type.